test/com/facebook/buck/apple/clang/UmbrellaHeaderTest.java

```diff
@@ -0,0 +1,15 @@
+package com.facebook.buck.apple.clang;
+
+import static org.junit.Assert.assertEquals;
+
+import com.google.common.collect.ImmutableList;
+import org.junit.Test;
+
+public class UmbrellaHeaderTest {
+
+  @Test
+  public void testUmbrellaHeader() {
+    UmbrellaHeader header = new UmbrellaHeader(ImmutableList.of("header1.h", "header2.h"));
+    assertEquals("#import \"header1.h\"\n" + "#import \"header2.h\"\n", header.render());
```

Comment:
This implies that umbrella headers are not exported as those imports would not work from other libraries. Do we plan to expose those umbrella headers, so that non-modular libraries can do `#import <Module/Module.h>`?

Reply:
My bad, I'll fix it to take a name and fix the headers.